[RHACS] [rhacs-docs-4.4] ROX23460 Adding 4.4.1 RN (#74570)
openshift/openshift-docs: release_notes/44-release-notes.adoc

+
This update ensures that the linked collection is retained, avoiding the error message.

[id="resolved-in-version-441_{context}"]
=== Resolved in version 4.4.1

*Release date*: 15 April 2024

* With this update, rendering the Helm charts `stackrox-central-services` and `stackrox-secured-cluster-services` requires a Helm version 3.9.0 or later, as support for older Helm versions has been discontinued.

> * Fixed an issue where {product-title-short} would not qualify the registry and path of images from the container runtime if the `ROX_UNQUALIFIED_SEARCH_REGISTRIES` environment variable was set to `true` on both Central and Sensor. This update now enables support for CRI-O's unqualified search registries and short name aliases.

Reviewer: I thought we do not document this - @dcaravel to verify whether this should be here or not.

Reviewer 2: That's correct, please remove the `ROX_UNQUALIFIED_SEARCH_REGISTRIES` section for now. Thanks @agantony

Author: ack

+
This update ensures that the linked collection is retained, avoiding the error message.

[id="resolved-in-version-441_{context}"]
=== Resolved in version 4.4.1

*Release date*: 15 April 2024

* Fixed an issue where rendering the Helm charts `stackrox-central-services` and `stackrox-secured-cluster-services` required a Helm version 3.9.0 or later, as support for older Helm versions has been discontinued.

* Fixed an issue where {product-title-short} would not qualify the registry and path of images from the container runtime if the `ROX_UNQUALIFIED_SEARCH_REGISTRIES` environment variable was set to `true` on both Central and Sensor. This update now enables support for CRI-O's unqualified search registries and short name aliases.

* Fixed an issue that prevented the filtering of newly added or updated policies by category by using the `roxctl` CLI.

[id="known-issues-440_{context}"]
== Known issues

*Updated on*: 15 April 2024

> * Currently, there is an issue for OpenShift Container Platform 4.12 on the Power architecture where Collector uses the EBPF collection method when CORE_BPF is selected from {product-title-short} 4.4.1 and later.
> +
> Workaround: If you use OpenShift Container Platform 4.12 Power offline, you must use a support package.

Reviewer 3: ```suggestion
* Currently, there is an issue for OpenShift Container Platform 4.12 on the Power architecture where CORE_BPF collection method is not functioning properly. From {product-title-short} 4.4.1 and later, in this particular case, Collector uses automatically the EBPF collection method when CORE_BPF is selected.

Impact: If you use OpenShift Container Platform 4.12 on the Power architecture and in offline mode, you must use a support package.
```

+
This update ensures that the linked collection is retained, avoiding the error message.

[id="resolved-in-version-441_{context}"]
=== Resolved in version 4.4.1

*Release date*: 15 April 2024

* Fixed an issue that prevented the filtering of newly added or updated policies by category by using the `roxctl` CLI.

[id="known-issues-440_{context}"]
== Known issues

*Updated on*: 15 April 2024

> * Currently, there is an issue for OpenShift Container Platform 4.12 on the Power architecture where the CORE_BPF collection method does not work properly. From {product-title-short} 4.4.1 and later, Collector automatically uses the EBPF collection method in this particular case when CORE_BPF is selected. 

Reviewer 4: ```suggestion
* Currently, there is an issue for OpenShift Container Platform 4.12 on the IBM Power architecture where the CORE_BPF collection method does not work properly. From {product-title-short} 4.4.1 and later, Collector automatically uses the EBPF collection method in this particular case when CORE_BPF is selected. 
```

> If you are using OpenShift Container Platform 4.12 on the Power architecture and in offline mode, you must use a support package.

Reviewer 4: ```suggestion
If you are using OpenShift Container Platform 4.12 on the IBM Power architecture and in offline mode, you must use a support package.
```
